git/git.go

```diff
@@ -298,3 +291,12 @@ func firstLine(output []byte) string {
 	}
 	return string(output)
 }
+
+func getBranchShortName(output []byte) string {
+	branch := firstLine(output)
+	prefix := "refs/heads/"
+	if i := strings.Index(branch, prefix); i != -1 {
+		return branch[i+len(prefix):]
```

Comment:
Tip: this can be simplified with `strings.TrimPrefix(t, "refs/heads/")` and is also more exact, since we only every want to strip this prefix and not when it appears in the middle (e.g. `crossrefs/heads/foo` is a valid branch name)

Reply:
thanks, no need to reinvent the wheel 👍🏻